Bundles/ProductMerchantPortalGui/src/Spryker/Zed/ProductMerchantPortalGui/Persistence/ProductMerchantPortalGuiRepository.php

```diff
@@ -379,8 +379,14 @@ protected function addProductAbstractSorting(
         SpyMerchantProductAbstractQuery $merchantProductAbstractQuery,
         MerchantProductTableCriteriaTransfer $merchantProductTableCriteriaTransfer
     ): SpyMerchantProductAbstractQuery {
-        $orderColumn = $merchantProductTableCriteriaTransfer->getOrderBy() ?? static::COL_KEY_PRODUCT_ABSTRACT_SKU;
         $orderDirection = $merchantProductTableCriteriaTransfer->getOrderDirection() ?? Criteria::DESC;
+        if ($merchantProductTableCriteriaTransfer->getOrderBy() === null) {
+            $merchantProductAbstractQuery->orderByIdMerchantProductAbstract($orderDirection);
+
+            return $merchantProductAbstractQuery;
+        }
+
+        $orderColumn = $merchantProductTableCriteriaTransfer->getOrderBy();
 
         if (!$orderColumn || !$orderDirection) {
```

Comment:
mb we can also update this part? I mean, there is no way now that $orderColumn is empty